vsphere/datadog_checks/vsphere/legacy/vsphere_legacy.py

```diff
@@ -917,14 +905,8 @@ def collect_metrics(self, instance):
                     max_historical_metrics = int(vcenter_settings[0].value)
                 if max_historical_metrics < 0:
                     max_historical_metrics = float('inf')
-            except Exception as e:
-                self.log.debug(
-                    "Error getting maxQueryMetrics setting "
-                    "(max_historical_metrics=%s, DEFAULT_MAX_HIST_METRICS=%s): %s",
-                    max_historical_metrics,
-                    DEFAULT_MAX_HIST_METRICS,
-                    e,
-                )
```

Comment:
Same.

Can we move the changes of the legacy implementation to another separate PR?